### PR TITLE
AST-2253 Fixed admin dashboard redesign issues with layouts

### PR DESCRIPTION
**Pull request by er-roshan**
### Description
AST-2253 Fixed admin dashboard redesign issues with layouts

### Types of changes
Bug fix (non-breaking change which fixes an issue)

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: admin/assets/src/dashboard-app/pages/custom-layouts/CustomLayouts.js [hunk 1]
```diff
@@ -38,15 +38,23 @@ const CustomLayouts = () => {
 								className="text-2xl font-semibold leading-7 text-slate-800"
 								id="modal-title"
 							>
-								Upgrade to Pro
+								{ astra_admin.upgrade_notice ? 'Upgrade to Pro' : 'Custom Layouts' }
```
Comment (imnavanath) [inline admin/assets/src/dashboard-app/pages/custom-layouts/CustomLayouts.js:41]: translation missing

File: admin/assets/src/dashboard-app/pages/custom-layouts/CustomLayouts.js [hunk 2]
```diff
@@ -38,15 +38,23 @@ const CustomLayouts = () => {
...
 							<div className="mt-2">
-								<p className="text-sm text-slate-500">
-									Get access to powerful features for painless
-									WordPress designing, without the high costs.
-									With all the time you will save, it’s a
-									product that pays for itself!
-								</p>
+								{ astra_admin.upgrade_notice ?
+									<p className="text-sm text-slate-500">
+										Get access to powerful features for painless
```
Comment (imnavanath) [inline admin/assets/src/dashboard-app/pages/custom-layouts/CustomLayouts.js:46]: translation missing

File: admin/assets/src/dashboard-app/pages/custom-layouts/CustomLayouts.js [hunk 3]
```diff
@@ -38,15 +38,23 @@ const CustomLayouts = () => {
...
+										WordPress designing, without the high costs.
+										With all the time you will save, it’s a
+										product that pays for itself!
+									</p> :
+									<p className="text-sm text-slate-500">
+										Custom Layouts is a very powerful module that comes with Astra Pro.
```
Comment (imnavanath) [inline admin/assets/src/dashboard-app/pages/custom-layouts/CustomLayouts.js:52]: translation missing

File: admin/assets/src/dashboard-app/pages/custom-layouts/CustomLayouts.js (hunk 1 above)
Comment (er-roshan) [reply admin/assets/src/dashboard-app/pages/custom-layouts/CustomLayouts.js]: I have fixed the missing translation

File: admin/assets/src/dashboard-app/pages/custom-layouts/CustomLayouts.js (hunk 2 above)
Comment (er-roshan) [reply admin/assets/src/dashboard-app/pages/custom-layouts/CustomLayouts.js]: I have fixed the missing translation

File: admin/assets/src/dashboard-app/pages/custom-layouts/CustomLayouts.js (hunk 3 above)
Comment (er-roshan) [reply admin/assets/src/dashboard-app/pages/custom-layouts/CustomLayouts.js]: I have fixed the missing translation

**Review by imnavanath — APPROVED** [review]

**Pull request merged**